firefox-ios/Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -227,7 +225,7 @@ class TabManagerImplementation: NSObject, TabManager, FeatureFlaggable {
     }
 
     func addNavigationDelegate(_ delegate: WKNavigationDelegate) {
```

Comment:
nit; since we only have one navigation delegate, this could be `set` instead of `add`?
```suggestion
    func setNavigationDelegate(_ delegate: WKNavigationDelegate) {
```

Reply:
yes i agree, it is not misleading with the set